__test__/base_decorator.test.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../nameable'
+require_relative '../decorators/base_decorator'
+
```

Comment:
- [ ] Your tests do not run when I run the rspec command: `rspec __test__` on the root directory.

  I fixed this by renaming all test files so that they end with `_spec.rb`, instead of `.test.rb`. I don't think rspec recognizes `.test.rb` files. It checks for `spec.rb`

  Example: `base_decorator_spec.rb`, `book_spec.rb`, `classroom_spec.rb`, etc.

  ![image](https://github.com/SamTush/school-library/assets/81468675/3e2a641a-406c-4681-94e1-7602e02f3708)

- [ ] **[OPTIONAL]** Rspec test files in ruby and rails are often placed in `spec` folders, not `__test__`. I think the latter convention is for Javascript and React projects.